lib/takeaway.rb

```diff
@@ -1,13 +1,16 @@
 require_relative 'dish'
+require_relative 'twilioruby'
+require 'env'
 
 class Takeaway
 
   attr_reader :menu, :my_choices
 
   def initialize
     @menu_object = []
-    @menu = {"Bryani" => 7, "Curry" => 9, "Beef" => 10, "Shrimp" => 11, "Korma" => 4}
```

Comment:
Maybe could be done as an array of hashes instead of just a hash. But this is just a suggestion.